docs/product_search/index.md

```diff
@@ -3,4 +3,82 @@ layout: default
 title: Product Search
 has_children: true
 nav_order: 8
+has_toc: false
 ---
+
+# Product Search
+Product search is based on query parameters that determine which products are returned and in which order. In addition to the search query itself, there are parameters that allow you to customize the search further, for example you could:
```

Comment:
Firstly I wouldn't call them query parameters here, just parameters generally. Query parameters are added/taken from the url. 
I'd also just say that the parameters define filters and sorting, and that we have lots of Easol specific/custom filters. Otherwise it's not clear to me what distinction we're drawing between 'search query itself' and further custom searching.

Reply:
Good point, will make this more clear